tools/topology/topology2/include/pipelines/cavs/host-copier-micsel-gain-mixin-playback.conf

```diff
@@ -0,0 +1,361 @@
+#
+# FE playback pipeline: host-copier-micsel-gain-mixin-playback
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.host-copier-micsel-gain-mixin-playback.N.attribute_name"
+#
+# Usage: host-copier-micsel-gain-mixin-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.host-copier-micsel-gain-mixin-playback."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/input_audio_format.conf>
+<include/common/output_audio_format.conf>
+<include/components/host-copier.conf>
+<include/components/micsel.conf>
+<include/components/mixin.conf>
+<include/components/pipeline.conf>
+<include/components/gain.conf>
+
+Class.Pipeline."host-copier-micsel-gain-mixin-playback" {
```

Comment:
@ranj063 I wasn't able to use here the subtree copy from host-copier-gain-mixin-playback pipeline class. The topology built but the formats were not updated correctly for host-copier, so non-2ch playback failed. Could well be my bad but I can retry after vacation.